source/layouts/layout.erb

```diff
@@ -97,7 +97,7 @@
       <hr>
       <!-- footer -->
       <div class="footer pull-right">
-        <p>&copy; 2014 <a href="https://twitter.com/">Twitter, Inc.</a> and other contributors
+        <p>&copy; 2016 <a href="https://twitter.com/">Twitter, Inc.</a> and other contributors
```

Comment:
Pretty sure these should be 201x-2016 rather than a specific year or so a lawyer told me once. :-)